src/pages/home/report/ReportActionsView.tsx

```diff
@@ -148,32 +144,85 @@ function ReportActionsView({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [route, isLoadingInitialReportActions]);
 
+    // Get a sorted array of reportActions for both the current report and the transaction thread report associated with this report (if there is one)
+    // so that we display transaction-level and report-level report actions in order in the one-transaction view
+    const combinedReportActions = useMemo(() => {
+        if (isEmptyObject(transactionThreadReportActions)) {
```

Comment:
In case the thread report actions aren't fetched yet, it causes an issue https://github.com/Expensify/App/issues/43034. So instead of actions we can validate only transactionThreadReportID.